### PR TITLE
Improve extract_za_id_dob

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
If this extracted part is 540529 for example, then the function returns 2054-05-29 which is wrong by a century.

in old code we did: ```// override moment default century switch at '68 with '49
moment.parseTwoDigitYear = function (input) {
    return +input + (+input > 49 ? 1900 : 2000);
};```

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr @imsickofmaps please review

File: test/test_utils.js [hunk 1]
```diff
@@ -321,6 +321,10 @@ describe("Testing utils functions", function() {
             // 'Invalid date' when input length < 4
             assert.deepEqual(utils.extract_za_id_dob("810"), "Invalid date");
         });
+        it("", function() {
+            assert.deepEqual(utils.extract_za_id_dob("5202017805280"),
+                moment("1952-02-01").format("YYYY-MM-DD"));
+        });
```
Comment (gsvr) [inline test/test_utils.js]: Please add more tests, especially at the boundaries - like last day of 1948, first day of 1949, where the boundary used to be (around 1968), and year 2000.

**Comment by NicolaasDieBaas** [comment]
@gsvr 2nd pass review please
